src/aggregator/integration/integration.go

```diff
@@ -36,3 +38,34 @@ func waitUntil(fn conditionFn, timeout time.Duration) bool {
 	}
 	return false
 }
+
+// aggregatedMetric is useful for JSON comparison of expected metrics.
+type aggregatedMetric struct {
+	ID            string
+	Type          string
+	Time          time.Time
+	Value         float64
+	StoragePolicy string
+}
+
+func newAggregatedMetric(
+	m aggregated.MetricWithStoragePolicy,
+) aggregatedMetric {
+	return aggregatedMetric{
+		ID:            m.Metric.ID.String(),
+		Type:          m.Metric.Type.String(),
+		Time:          time.Unix(0, m.Metric.TimeNanos),
```

Comment:
Worth considering a string formatted version of this for easier visual comparison?

Reply:
This actually prints as a string in JSON thankfully 👍